packages/sdk-v2/src/sdk/goodid/utils/resolverAgent.ts

```diff
@@ -0,0 +1,33 @@
+// import { createAgent, IResolver } from "@veramo/core";
```

Comment:
remove this. resolver/verifier is located server side

Reply:
removed